net-setup.sh

```diff
@@ -62,6 +62,7 @@ if [ `id -u` != 0 ]; then
 fi
 
 IFACE=zeth
+PIDFILE="/tmp/.${IFACE}.pidfile"
 
```

Comment:
`nat.conf` and `nat.conf.stop` already use different way to starting, keeping the pidfile and stopping.